modules/googleplay/src/test/scala/cards/nine/googleplay/service/free/interpreter/cache/MockInterpreter.scala

```diff
@@ -32,19 +32,17 @@ trait InterpreterServer[F[_]] {
   def listPending(num: Int): F[List[Package]]
 }
 
-case class MockInterpreter[F[_]](server: InterpreterServer[F]) extends (Ops ~> F) {
+case class MockInterpreter[F[_]](server: InterpreterServer[F]) extends Handler[F] {
```

Comment:
Can we say `Cache.Handler[F]`?

We can also remove the `cats.~>` import, right?